### DISCUSSION TITLE
RSL Helper outdated / link blocked

### DISCUSSION THREAD

**Discussion by madam-mim86** [FarbstoffRSL/RSL-Helper#453] [General]
Whenever I open up RSL Helper these last couple of days, it says that it needs to be updated and shuts down.  I use the updater in administrator mode and it says there are no updates.  I thought I would delete and just reinstall to see if that fixed it.  The bitly link to the download has now been blocked.  

![Screenshot 2022-09-30 123119](https://user-images.githubusercontent.com/114766965/193325113-d515efd4-16db-40d3-af51-6970787e7cee.jpg)

**Comment by connormiddleton1** [comment]
Yeah same for me
I really wanted this and I might have a solution i will keep you updated to be able to get it i just need to see if it works
But yeah my friends have been using it for a while it is definitly not malicious

**Reply by madam-mim86** [reply]
How To find the download for the rslhelper:

1. Download the last initial setup or full setup (https://installer.rsl-helper.de/). If you run in secure problems, try to change https to http.
2. Install it.
3. After the install you find a rslhelperupdater.exe in the folder of the helper (standard is c:\program files\RSL_Helper_X64). Start it with admin rights. Than you get the last version of the helper.

**Comment by Fredtals** [comment]
Any news regarding this? The link is still considered as malicious by Bitly...

**Reply by madam-mim86** [reply]
From the Discord Server:

How To find the download for the rslhelper:

1. Download the last initial setup or full setup (https://installer.rsl-helper.de/). If you run in secure problems, try to change https to http.
2. Install it.
3. After the install you find a rslhelperupdater.exe in the folder of the helper (standard is c:\program files\RSL_Helper_X64). Start it with admin rights. Than you get the last version of the helper.
Hint: If the updater is not updating, look in pinned messages for further instructions. It is written in it.

**Reply by neseforov** [reply]
Hello
Same problem. I do everything according to the instructions - an error even after the update. How to fix?
![image](https://user-images.githubusercontent.com/42833684/224488648-a5a75aee-2574-48bc-99c2-9f51897ef4d4.png)
